### PR TITLE
Minor dionae changes

### PR DESCRIPTION
**Pull request by Yonnimer**
Just adds the eternal as a religion for dionae to choose, and adds a sprite to go along with it

Also changes the blurb for dionae to be more up to date.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: aurorastation.dme [hunk 1]
```diff
@@ -1254,6 +1254,7 @@
 #include "code\modules\background\religion\tajara.dm"
 #include "code\modules\background\religion\unathi.dm"
 #include "code\modules\background\religion\vaurca.dm"
+#include "code\modules\background\religion\diona.dm"
```
Comment (JohnWildkins) [inline aurorastation.dme:1257]: Looks good, but please put the #include for diona.dm above the other religion includes (in alphabetical order) or else compilation and bug torture will ensue

**Review by JohnWildkins — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Chada-1 — APPROVED** [review]

**Review by mikomyazaki — APPROVED** [review]

File: aurorastation.dme (hunk 1 above)
Comment (Yonnimer) [reply aurorastation.dme]: Should be fixed now.

Comment (Yonnimer) [reply aurorastation.dme]: Should /actually/ be fixed after I had a small brain moment

**Pull request merged**